### ISSUE TITLE
Add examples to the partitionkey extension

### ISSUE THREAD

**Issue by duglin** [cloudevents/spec#429]
See: https://github.com/cloudevents/spec/pull/218

@jroper @clemensv - could one of you consider taking care of this one?

**Comment by duglin** [comment]
Clearly not required for v1.0 but is this something that someone would like to try to get added during our "spec review" period? Extra docs are always a good thing :-)

@clemens @bluemonk3y  ?

**Comment by clemens** [comment]
This is not the Clemens you're looking for. ;)

**Comment by duglin** [comment]
LOL sorry - @clemens 

@clemensv  see above ^^^

**Comment by fnobilia** [comment]
Is it still needed? I'm happy to give it a try!

**Comment by MagicJohnJang** [comment]
How to use partition key on v2 ?

**Comment by MagicJohnJang** [comment]
NM,  I got it here :
https://github.com/cloudevents/sdk-csharp/blob/1b9c7cc8c790eb89dd825dd8b50794b49e6a4018/test/CloudNative.CloudEvents.UnitTests/Extensions/PartitioningTest.cs#L51

**Comment by duglin** [comment]
This issue is running the risk of being closed soon. If you'd like to volunteer to drive it please speak up soon!

**Comment by duglin** [comment]
@xibz will take a look at this one...will revisit again soon

**Comment by jskeet** [comment]
Suggest we close unless we get a volunteer to do this imminently.

**Comment by duglin** [comment]
@xibz did you still want to find time to work on this one?

**Comment by xibz** [comment]
unfortunately no. Been too busy with verifiability. I'd opt to find another owner at this point or close

**Comment by duglin** [comment]
Agreed to close on the 2/19 call - if someone wants to do this ping us and we'll reopen it
